{
    "crime": "4825. Ljava/net/ConnectException;<init>->Ljava/lang/AssertionError;getCause",
    "permission": [],
    "api": [
        {
            "class": "Ljava/net/ConnectException;",
            "method": "<init>",
            "descriptor": "(Ljava/lang/String;)V"
        },
        {
            "class": "Ljava/lang/AssertionError;",
            "method": "getCause",
            "descriptor": "()Ljava/lang/Throwable;"
        }
    ],
    "score": 1,
    "label": []
}